### PR TITLE
Update UI when dragging a folder

### PR DESCRIPTION
**Pull request by nikhilweee**
Hello!

First of all, thank you for such an amazing tool. This has been a life saver for me. I loved the functionality to drag and drop folders directly in the browser, and this makes it a truly portable solution without me having to install anything.

This all worked fine, however, I noticed that when my folder had multiple URDF files, I was confused as to which one was being loaded. In such cases, I would love the choice to select a model of my choice, and that's exactly what is being addressed in this PR. I'm open to any changes you might suggest.

Thanks,
Nikhil

### CODE REVIEW

File: javascript/example/src/dragAndDrop.js [hunk 1]
```diff
@@ -157,14 +157,26 @@ export function registerDragEvents(viewer, callback) {
                 viewer.up = '+Z';
                 document.getElementById('up-select').value = viewer.up;
 
+                const availableModels = fileNames.filter(n => /urdf$/i.test(n));
+                const urdfOptions = document.querySelector('#urdf-options');
+                while( urdfOptions.firstChild ){
```
Comment (gkjohnson) [inline javascript/example/src/dragAndDrop.js:162]: nit: can we remove the spaces in the parens so it matches the rest of the code.

**Review by gkjohnson — CHANGES_REQUESTED** [review]
Great! Thanks for the contribution! This is a great addition. A few more requests beyond the comments:

- Can we revert or remove the bundled script files from the PR? They are now automatically generated via CI. I'd actually prefer if you just delete them entirely since they're no longer needed in the master branch.
- Can we enable a scroll bar on the models list to show at most 4 or so model names? Right now you can drag a folder with dozens of files and it scrolls off the page:

image

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 URDF loading code in both [C# for Unity](./unity/Assets/URDFLoader/) and [Javascript for THREE.js](./javascript/), as well as example [JPL ATHLETE](https://en.wikipedia.org/wiki/ATHLETE) URDF files
 
-[Demo Here!](https://gkjohnson.github.io/urdf-loaders/javascript/example/bundle/index.html)
+[Demo Here!](https://nikhilweee.me/urdf-loaders/javascript/example/bundle)
```
[the remark this answers is not in the document]
Comment (nikhilweee) [reply README.md:5]: Oh yes sorry didn't realise the changes would reflect here too.

**Comment by nikhilweee** [comment]
@gkjohnson Let me know if this works!

**Comment by gkjohnson** [comment]
Excellent, thank you!

**Pull request merged**